src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -55,8 +55,9 @@ final class ProgressBar
 
     /**
      * @param int $max Maximum steps (0 if unknown)
+     * @param float $minSecondsBetweenRedraws Minimum time between redraws (defaults to 60fps)
```

Comment:
this should be removed IMHO, it doesn't add enough value

Reply:
sure, can remove it again. Should I also remove the line above then?